beetsplug/fetchart.py

```diff
@@ -192,6 +192,64 @@ def get(self, album):
                 yield item['link']
 
 
+class FanartTV(ArtSource):
+    """Art from fanart.tv requested using their API"""
+    API_URL = 'http://webservice.fanart.tv/v3/'
+    API_ALBUMS = API_URL + 'music/albums/'
+    PROJECT_KEY = ''
+
+    def get(self, album):
+        if not album.mb_releasegroupid:
+            return
+
+        response = self.request(
+            self.API_ALBUMS + album.mb_releasegroupid,
+            headers={
+                'api-key': self.PROJECT_KEY,
+                'client-key': self._config['fanarttv_key'].get()
+            })
+
+        try:
+            data = response.json()
+        except ValueError:
+            self._log.debug(u'fanart.tv: error loading response: {}',
+                            response.text)
+            return
+
+        def escape_for_log(s):
+            # the logger will eventually try to .format() the message, and
+            # interpret the dict as format spec...
+            return ''.join((2 * c if c in '{}' else c for c in s))
+        self._log.debug(escape_for_log(str(data)))
```

Comment:
The usual way to do this is just to insert the string using the formatter:

    self._log.debug(u'fanart.tv: response: {}', data)

Or, in this case, you could enhance readability by doing:

    self._log.debug(u'fanart.tv: response: {}', json.dumps(data, indent=2, sort_keys=True))